frontend/admin/src/js/components/apiManagedTable/TableHeader.tsx

```diff
@@ -52,135 +52,128 @@ function TableHeader<T extends Record<string, unknown>>({
-          data-h2-radius="b(s, s, none, none)"
-          data-h2-padding="b(all, s)"
-        >
-          <div style={{ flexShrink: 0 }}>
-            {title && <span data-h2-font-weight="b(800)">{title}</span>}
-          </div>
-          <div
-            style={{ flexShrink: 0 }}
-            data-h2-display="b(flex)"
-            data-h2-justify-content="b(flex-end)"
-          >
-            <SearchForm onChange={onSearchChange} searchBy={searchBy} />
-            <Spacer>
-              <Button
-                mode="outline"
-                color="black"
-                type="button"
-                data-h2-display="b(inline-flex)"
-                data-h2-align-items="b(center)"
-              >
-                <ButtonIcon icon={FilterIcon} />
-                <span>
-                  {intl.formatMessage({
-                    defaultMessage: "Filters",
-                    description:
-                      "Text label for button to open filter dialog on admin tables.",
-                  })}
-                </span>
-              </Button>
-            </Spacer>
-            <Spacer>
-              <div data-h2-position="b(relative)">
-                <Button
-                  mode="outline"
-                  color="black"
-                  type="button"
-                  data-h2-display="b(inline-flex)"
-                  data-h2-align-items="b(center)"
-                  onClick={() => setShowList(!showList)}
-                >
-                  <ButtonIcon icon={TableIcon} />
-                  <span>
-                    {intl.formatMessage({
-                      defaultMessage: "Columns",
-                      description:
-                        "Label displayed on the Table Columns toggle button.",
-                    })}
-                  </span>
-                </Button>
-                <Dialog
-                  color="ts-primary"
-                  isOpen={showList}
-                  onDismiss={() => setShowList(false)}
-                  title={intl.formatMessage({
-                    defaultMessage: "Table columns",
-                    description:
-                      "Dialog title for the admin tables columns toggle.",
-                  })}
-                >
-                  <Fieldset
-                    legend={intl.formatMessage({
-                      defaultMessage: "Visible columns",
-                      description:
-                        "Legend for the column toggle in admin tables.",
-                    })}
+        <div data-h2-margin="base(x2, 0, x.5, 0)">
+          <p>{title && <span data-h2-font-weight="base(700)">{title}</span>}</p>
+          <div data-h2-flex-grid="base(center, 0, x1)">
+            <div data-h2-flex-item="base(fill)">
+              <div data-h2-flex-grid="base(center, 0, x.5)">
+                <div data-h2-flex-item="base(content)">
+                  <SearchForm onChange={onSearchChange} searchBy={searchBy} />
```

Comment:
Search form is being overlapped by the "New X" button.

<img width="465" alt="Screen Shot 2022-08-18 at 10 58 57 AM" src="https://user-images.githubusercontent.com/4127998/185445730-0cf253cd-0d8a-46ac-9c88-f52b2743b062.png">